frmts/jp2kak/jp2kakdataset.cpp

```diff
@@ -2869,6 +2942,24 @@ void GDALRegister_JP2KAK()
 "   <Option name='FLUSH' type='boolean' />"
 "   <Option name='NBITS' type='int' description="
 "'BITS (precision) for sub-byte files (1-7), sub-uint16 (9-15)'/>"
+"   <Option name='RATE' type='string' description='One or more bit-rates, expressed in terms of the ratio between"
```

Comment:
the description is untypically long here. This longer description should be moved to the Creation Options section of doc/source/drivers/raster/jp2kak.rst

Reply:
Fix in 7ec84bcd0aea12104456695e6ce03386c52c7455

Reply:
You have forgotten to commit the changes in doc/source/drivers/raster/jp2kak.rst to describe the 2 new options.

Reply:
Fix in c2275ddf75382210b2393b457b25ec6fee8f0752